src/com/Dryft/DAOs/UserDAO.java

```diff
@@ -0,0 +1,58 @@
+package com.Dryft.DAOs;
+
+import java.sql.*;
+import com.Dryft.models.User;
+import com.Dryft.utils.DBConn;
+import com.Dryft.exceptions.UserSideException;
+
+public class UserDAO {
+
+    public static void createUser(User user) throws Exception {
+        conn = DBConn.getConn();
+        checkUserAlreadyExists(user, conn);
+        String query = "Insert into users values (?,?,?,?,?,?)";
+        PreparedStatement st = conn.preparetatement(query);
+        st.setString(6, user.getEmail(), user.getFullname(), user.getPassword(), user.getSalt(), user.getSex(),
+                user.getBalance());
+        ResultSet result = st.executeQuery();
+        DBConn.closeConn();
+    }
+
+    private static void checkUserAlreadyExists(User user, DBconn conn) throws Exception {
```

Comment:
You dont need to write the throws part, I think. Or if you do, write the full name of the exception

Reply:
I do need to write the throws part as I am not handling the exceptions inside each method.
Moreover,There can be SQL exceptions also.Either extend the UserSideException class from the SQLException ,then I will write the complete UserSide Exception.Or I write throws SQLExceptions,UserSideExceptions. Which one do you prefer?

Reply:
explicit is better than implicit